src/Client.php

```diff
@@ -96,6 +102,7 @@ public function __construct(array $config = [])
         $this->secret = $config['secret'];
         $this->partnerId = $config['partner_id'];
         $this->shopId = $config['shopid'];
```

Comment:
Should be `shop_id` instead of `shopid` for naming consistency?

Reply:
i've done